portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
```

Comment:
Does this line still apply?

Reply:
to clarify: line 13

Reply:
I was going to change it to something similar in a future commit but just saw your comment below on deleting commented code and accessing it via the history so I will do that instead :)